### PR TITLE
Abhijit_Assignment_8_Done

### PR DESCRIPTION
**Pull request by AbhijitMJun20**
Assignment_8_Done

### CODE REVIEW

File: src/abhijitMJun20/Bankingsystem.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package abhijitMJun20;
+
+public class Bankingsystem {
+	int accountbalance=1000;
```
Comment (KrishnaKTechnocredits) [inline src/abhijitMJun20/Bankingsystem.java]: No Need to initialize this variable. perform credit() method first.

File: src/abhijitMJun20/Bankingsystem.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	static int credittransactionsummary=0,debittransactionsummary=0,printbalancesummary=0;
+	int creditcount=0,debitcount=0,printBalancecount=0;
```
Comment (KrishnaKTechnocredits) [inline src/abhijitMJun20/Bankingsystem.java]: Remove the initialization of all instance variables.

File: src/abhijitMJun20/Bankingsystem.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	
+	void debitAmount(int debitamount) { 
+		accountbalance=accountbalance-debitamount;
```
Comment (KrishnaKTechnocredits) [inline src/abhijitMJun20/Bankingsystem.java:9]: You should write the condition to verify the insufficient balance.

File: src/abhijitMJun20/Bankingsystem.java (hunk 1 above)
Comment (AbhijitMJun20) [reply src/abhijitMJun20/Bankingsystem.java]: Done

File: src/abhijitMJun20/Bankingsystem.java (hunk 2 above)
Comment (AbhijitMJun20) [reply src/abhijitMJun20/Bankingsystem.java]: Done

File: src/abhijitMJun20/Bankingsystem.java (hunk 3 above)
Comment (AbhijitMJun20) [reply src/abhijitMJun20/Bankingsystem.java:9]: Done

**Pull request merged**